server/controllers/reviewDataConroller/reviewDataRepository.js

```diff
@@ -0,0 +1,38 @@
+const { ReviewData } = require('../../db/models/Review');
+
+class ReviewDataRepository {
```

Comment:
I love how you are using a class to structure out your CRUD functionality. You have some great code here, and I can already tell I'm gonna learn a ton from you. I can't wait for the next one.

Reply:
Thank you sir! Looking forward to reading your code as well.